pagopa/io-app
# feat(IDPay): [IODPAY-162] Add instruments status polling in initiative configuration steps

## Short description
This PR adds the polling mechanism in the initiative instrument configuration steps.

## List of changes proposed in this pull request
- Added sub-states to trigger instruments refresh after some delay (5000 ms)

## How to test
- Open the details screen of an IDPay initiative 
- Go to the instrument configuration screens
- Check that the app is polling the backend with an inspector tool.
